modules/oadp-upgrading-oadp-operator-1-5-0.adoc

```diff
@@ -0,0 +1,27 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="oadp-upgrading-dpa-operator-1-5-0_{context}"]
+= Upgrading the OADP Operator
+
+Use the following procedure when upgrading the {oadp-first} Operator.
```

Comment:
Use something like "You can upgrade the...."

> Avoid self-referential language, such as "This topic covers…​" or "Use this procedure to…​".
- https://redhat-documentation.github.io/supplementary-style-guide/#shortdesc